Engine/src/Engine/EngineUI/UserInterface.cpp

```diff
@@ -10,14 +10,14 @@ namespace Engine
 	std::unordered_map<std::string, ImFont*> UserInterface::s_fonts;
 
 	//define the ImGUI vec4s for each colour
-	ImVec4 UserInterface::red = ImVec4(1.0f, 0.0f, 0.0f, 1.0f);
-	ImVec4 UserInterface::green = ImVec4(0.0f, 1.0f, 0.0f, 1.0f);
-	ImVec4 UserInterface::darkGreen = ImVec4(0.0f, 0.75f, 0.0f, 1.0f);
-	ImVec4 UserInterface::blue = ImVec4(0.0f, 0.0f, 1.0f, 1.0f);
-	ImVec4 UserInterface::grey = ImVec4(0.75f, 0.75f, 0.75f, 1.0f);
+	ImVec4 UserInterface::lightGrey = ImVec4(0.75f, 0.75f, 0.75f, 1.0f);
```

Comment:
If plausible, these colour variables should be consts and following the corresponding naming conventions.

Reply:
Can be const ImVec4 btw. Can be complicated implementing cpp variables as const when they're also stored as const in the header.

Reply:
changed colours to const and updated names